docs/content/guides/columns/column-filter.md

```diff
@@ -155,765 +450,2860 @@ import 'handsontable/dist/handsontable.full.min.css';
+        },
+        {
+          brand: 'Chatterpoint',
+          model: 'Road Tire Tube',
+          price: 59,
+          sellDate: 'Aug 28, 2023',
+          sellTime: '08:01 AM',
+          inStock: true,
+        },
+        {
+          brand: 'Eidel',
+          model: 'HL Road Tire',
+          price: 279.99,
+          sellDate: 'Oct 2, 2023',
+          sellTime: '01:23 AM',
+          inStock: true,
+        },
+      ]}
+      columns={[
+        {
+          title: 'Brand',
+          type: 'text',
+          data: 'brand',
+        },
+        {
+          title: 'Model',
+          type: 'text',
+          data: 'model',
+        },
+        {
+          title: 'Price',
+          type: 'numeric',
+          data: 'price',
+          numericFormat: {
+            pattern: '$ 0,0.00',
+            culture: 'en-US',
+          },
+          className: 'htLeft',
+        },
+        {
+          title: 'Date',
+          type: 'date',
+          data: 'sellDate',
+          dateFormat: 'MMM D, YYYY',
+          correctFormat: true,
+          className: 'htRight',
+        },
+        {
+          title: 'Time',
+          type: 'time',
+          data: 'sellTime',
+          timeFormat: 'hh:mm A',
+          correctFormat: true,
+          className: 'htRight',
+        },
+        {
+          title: 'In stock',
+          type: 'checkbox',
+          data: 'inStock',
+          className: 'htCenter',
+        },
+      ]}
+      // enable the column menu
+      dropdownMenu={true}
+      // enable filtering
+      filters={true}
+      // to differentiate this example's CSS from other examples on this page
+      className="custom-filter-button-example-2"
+      height="auto"
+      licenseKey="non-commercial-and-evaluation"
+    />
+  );
+};
 
-  const selectEntry = (rowEntry) => {
-    if (selectedRowEntries.includes(rowEntry)) {
-      setSelectedRowEntries(selectedRowEntries.filter(entry => entry !== rowEntry));
+/* start:skip-in-preview */
+ReactDOM.render(<App />, document.getElementById('exampleCustomFilterButton2'));
+/* end:skip-in-preview */
+```
 
-    } else {
-      setSelectedRowEntries([...selectedRowEntries, rowEntry]);
-    }
+```css
+/* hide the column menu button by default */
+.custom-filter-button-example-2 .changeType {
+  visibility: hidden;
+}
+
+/* show the column menu button on hover */
+.custom-filter-button-example-2 .relative:hover .changeType {
+  visibility: visible;
+}
+```
+
+:::
+
+:::
+
+## Change the width of the filter menu
+
+To make the filter menu wider, you can change the width of the entire
+[column menu](@/guides/columns/column-menu.md). For that, override Handsontable's CSS by using this
+selector: `.htDropdownMenu table.htCore`.
+
+```css
+/* style the column menu */
+.handsontable .htDropdownMenu table.htCore {
+  width: 300px !important;
+}
+```
+
+## Exclude rows from filtering
+
+You can disable filtering for specific rows by changing their indexes. This lets you keep portions
```

Comment:
> You can disable filtering for specific rows by changing their indexes.

I don't know what that means. Always give enough context for freshers, ideally addressing pain and providing a solution with the clear benefit of using it.

WHY would I need to disable filtering for specific rows? Probably for the same reason you don't want to sort particular rows. I would make this section a copycat of https://handsontable.com/docs/react-data-grid/rows-sorting/#exclude-rows-from-sorting